dahu/core/app/scripts/views/workspace/actions/disappear.js

```diff
@@ -5,19 +5,30 @@
 define([
     'handlebars',
     'backbone.marionette',
+    // views
+    'views/workspace/actions/action',
     // templates
+    'text!templates/views/workspace/actions/action.html',
     'text!templates/views/workspace/actions/disappear.html'
 ], function(
     Handlebars,
     Marionette,
+    // views
+    ActionView,
     // templates
+    actionTemplate,
     disappearTemplate
 ) {
     
     /**
      * Disappear action view
      */
-    return Marionette.ItemView.extend({
-        template: Handlebars.default.compile(disappearTemplate)
+    return ActionView.extend({
+        disappearTemplate: Handlebars.default.compile(disappearTemplate),
+
+        initialize: function (options) {
+            ActionView.prototype.initialize.call(this, options);
+            Handlebars.registerPartial("disappear", this.templateDisappear);
```

Comment:
use Handlebars.registerPartial("actionParametersPartial", Handlebars.default.compile(disappearTemplate)) is lot more simple ;)